src/Extensions/Abstractions/SfConfigurationException.cs

```diff
@@ -1,21 +1,21 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-using System;
-using System.Globalization;
-
-namespace Microsoft.Omex.Extensions.Abstractions
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
What are the differences here? Whitespace, invisible chars? It's making the PR hard to review.

Reply:
Yes, for multiple files, VS forced me to normalize line endings

Reply:
I assume it would be the same for everybody so I've decided it's better to commit them.

Reply:
In the future please try to commit cosmetic changes as separate PRs.